src/components/AudioTrack/layouts/Full.vue

```diff
@@ -28,8 +28,10 @@
         </div>
       </div>
 
-      <!-- TODO: Download dropdown -->
-      <div class="right-content bg-pink h-12 w-30 rounded-sm ml-auto" />
+      <DownloadButton
+        class="right-content h-14 flex"
```

Comment:
```suggestion 
```

These don't seem to have any affect on the styling. Removing them doesn't change the appearance

Reply:
I agree. I thought that `left-content` and `right-content` were left in their for some styling later on, so I decided to leave it in.